src/handwritten/task.py

```diff
@@ -402,6 +402,83 @@ def is_task_done(self):
+            skip unsupported channels: specifies whether or not to skip channels that 
+                do not support calibration. If skip unsupported channels is TRUE, this
+                VI calibrates only supported channels. If FALSE, this VI calibrates the 
+                channels specified by channels. The default is FALSE.
+        """
+        self._interpreter.perform_strain_shunt_cal_ex(
+            self._handle, channel, shunt_resistor_value,
+            shunt_resistor_location, shunt_resistor_select,
+            shunt_resistor_source, skip_unsupported_channels)
+
+    def perform_bridge_shunt_cal(
+            self, channel="", shunt_resistor_value=100000,
+            shunt_resistor_location=0, shunt_resistor_select=0,
+            shunt_resistor_source=0, bridge_resistance=120,
+            skip_unsupported_channels=False):
+        """
+        Performs shunt calibration for the specified channels of the task. 
+        The instances of this polymorphic VI correspond to the type of bridge 
+        sensor. Refer to the calibration procedure for your module for detailed
+        calibration instructions.
+
+        Performs shunt calibration for the specified channels using a bridge sensor.
+
+        Args:
+            task/channel in: specifies name of the task or a list of virtual 
+                channels to which the operation applies. If you provide a
+                list of virtual channels, NI-DAQmx creates a task automatically. 
+            channels: specifies a subset of virtual channels in the task that you 
+                want to calibrate. Use this input if you do not want to calibrate
+                all the channels in the task or if some channels in the task measure
+                non-bridge-based sensors. If the input is empty, this VI attempts to 
+                calibrate all virtual channels in the task.
+            shunt resistor value: specifies the shunt resistance in ohms.
+            shunt resistor location: specifies the location of the shunt resistor.
+            shunt resistor select: specifies which shunt calibration switch to enable.
+            shunt resistor source: specifies which shunt to use.
+            bridge resistance: specifies the bridge resistance in ohms. A value of -1
+                means to use the nominal bridge resistance specified when you created
+                the virtual channel.
+            skip unsupported channels: specifies whether or not to skip channels that 
+                do not support calibration. If skip unsupported channels is TRUE, this
+                VI calibrates only supported channels. If FALSE, this VI calibrates the 
+                channels specified by channels. The default is FALSE.
+        """
+        self._interpreter.perform_bridge_shunt_cal_ex(
+            self._handle, channel, shunt_resistor_value,
+            shunt_resistor_location, shunt_resistor_select,
+            shunt_resistor_source, bridge_resistance,
```

Comment:
This should expect enums.

```suggestion
            shunt_resistor_location.value, shunt_resistor_select.value,
            shunt_resistor_source.value, bridge_resistance,
```

Reply:
BTW, the docstring can specify the enum type in parens. Long-term, I would prefer for this to use type hints, but the current codebase uses parens in the docstring.

Reply:
This still needs done.